installation/scripts/generate-installer-artifacts.sh

```diff
@@ -39,7 +39,7 @@ function copyKymaInstaller() {
         curl -L https://storage.googleapis.com/kyma-development-artifacts/${release}/kyma-installer-cluster.yaml -o kyma-installer.yaml
         curl -L https://storage.googleapis.com/kyma-development-artifacts/${release}/is-installed.sh -o ${ARTIFACTS_DIR}/is-kyma-installed.sh
     else
-        curl -L https://github.com/kyma-project/kyma/releases/download/${release}/kyma-installer-cluster.yaml -o kyma-installer.yaml
```

Comment:
Please refactor the script as now the line is the same as if the condition is true.


 ```
curl -L https://storage.cloud.google.com/kyma-prow-artifacts/${release}/kyma-installer-cluster.yaml -o kyma-installer.yaml
 if [[ $release == *PR-* ]] || [[ $release == *master* ]]; then
// is installed from artifacts
 else
// is installed from repo
fi
```